DEVELOPER_GUIDE.md

```diff
@@ -46,7 +64,7 @@ Run tests :
 
 ### Send Message using Telnet
```

Comment:
Instead of telnet should we mention about running OpenSearch in another terminal as this was written before the connection was established?